config.go

```diff
@@ -7,6 +7,10 @@ import (
 	"github.com/BurntSushi/toml"
 )
 
+type Background struct {
+	Image  string `toml:"image"`
```

Comment:
Make sure to enable `gofmt` or `goimports` in your IDE. This will automatically format your source to match the language specs.

Reply:
Ohh thank you, did not now about that functionality as of now

Reply:
Yeah, it's a godsent really! :smile: